akka-projection-kafka/src/main/scala/akka/projection/kafka/internal/KafkaOffsetStore.scala

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
I thought about implementing a `KafkaOffsetStore` that behaved in a similar way to offset committing in Kafka, but it would be a strange choice to use this over a database `OffsetStore`.  One exception would be if we want to enable exactly once message delivery semantics on the producer side of the Kafka projection impl.  We could enlist both the storage of `KafkaOffsetStore.saveOffset` and `SendProducer.send` in the same transaction.  

This could be addressed in a later phase of projections.

Reply:
Let's leave that for later. Create a ticket if there isn't one already.

Reply:
Agreed.  I'm letting the idea percolate a bit more before creating an issue.  I want to get consensus on this phase 1 PR first.

Reply:
https://github.com/akka/akka-projection/issues/94